masterlist.yaml

```
@@ -2670,6 +2670,11 @@ plugins:
           - 'Cathedral Weathers and Seasons'
           - '[Cathedral Weathers - Audio Overhaul Patch for Skyrim SE patch](https://www.nexusmods.com/skyrimspecialedition/mods/25420/)'
         condition: 'active("Cathedral Weathers.esp") and not active("AOS - Catherdral Weathers Patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Immersive Sounds - Compendium'
+          - '[Extensive AOS - ISC Patch](https://www.nexusmods.com/skyrimspecialedition/mods/25051)'
+        condition: 'active("Immersive Sounds - Compendium.esp") and not active("Extensive Audio Overhaul Skyrim - Immersive Sounds Patch.esp")'
```

I'd leave the check for the default patch in there as well, just in case they don't want to use the new patch.
```suggestion
        condition: 'active("Immersive Sounds - Compendium.esp") and not active("Extensive Audio Overhaul Skyrim - Immersive Sounds Patch.esp") and not active("Audio Overhaul Skyrim - Immersive Sounds Patch.esp")'
```
An inc or/and `*alreadyIncludedInX` mesage could be added to `Audio Overhaul Skyrim - Immersive Sounds Patch.esp`.
```yaml
  - name: 'Audio Overhaul Skyrim - Immersive Sounds Patch.esp'
    inc: [ 'Extensive Audio Overhaul Skyrim - Immersive Sounds Patch.esp' ]
    msg:
      - <<: *alreadyInX
        subs: [ 'Extensive Audio Overhaul Skyrim - Immersive Sounds Patch.esp' ]
        condition: 'active("Extensive Audio Overhaul Skyrim - Immersive Sounds Patch.esp")'
```